components/brave_new_tab_ui/widgets/ftx/components/index.tsx

```diff
@@ -122,9 +122,8 @@ class FTX extends React.PureComponent<Props, State> {
     const selectedAsset = this.props.ftx.assetDetail?.currencyName
     const { showContent, widgetTitle } = this.props
     // Only show back arrow to go back to opt-in view
-    const shouldShowBackArrow = !selectedAsset &&
-      this.props.ftx.currentView !== ViewType.OptIn &&
-      !this.props.ftx.isConnected
+    const shouldShowBackArrow = showContent && !selectedAsset &&
```

Comment:
IMO,  back arrow could be visible when current view is markets and in not connected state.
and I think back arrow would be helpful also when asset is selected like below.
With this user can go back to opt-in view easily. WDYT? @ryanml  @petemill 

<img width="306"  src="https://user-images.githubusercontent.com/6786187/120147545-c6cd8880-c221-11eb-9892-2d8b79fe6451.png">

```
    const shouldShowBackArrow = showContent &&
      this.props.ftx.currentView === ViewType.Markets &&
      !this.props.ftx.isConnected
```

Reply:
@simonhong, makes sense to me! I've updated this with:

```
--- a/components/brave_new_tab_ui/widgets/ftx/components/index.tsx
+++ b/components/brave_new_tab_ui/widgets/ftx/components/index.tsx
@@ -119,10 +119,9 @@ class FTX extends React.PureComponent<Props, State> {
   }
 
   renderTitle () {
-    const selectedAsset = this.props.ftx.assetDetail?.currencyName
     const { showContent, widgetTitle } = this.props
     // Only show back arrow to go back to opt-in view
-    const shouldShowBackArrow = showContent && !selectedAsset &&
+    const shouldShowBackArrow = showContent &&
       this.props.ftx.currentView !== ViewType.OptIn
```

Though I omitted the check for `!this.props.ftx.isConnected`, as I believe we still want it shown in the connected state as well. Let me know if that isn't the case, thanks!

Reply:
Ah, or it could be that we don't want it to show when connected because at that point, the user should be disconnecting the widget as opposed to backtracking to pre-optin via the back arrow

Reply:
I've re-updated this with your suggested snippet 👍